tools/bridge/bridge/webservice.py

```diff
@@ -0,0 +1,89 @@
+import logging
+import os
+
+import falcon
+import pkg_resources
+from gevent.pywsgi import WSGIServer
+
+from bridge.service import Service
+
+logger = logging.getLogger(__name__)
+
+welcome_page = """
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<title>tlbc-bridge</title>
+</head>
+
+<body>
+<header>
+<h1>Welcome to tlbc-bridge</h1>
+</header>
+
+<p>
+You have reached tlbc-bridge's REST server. This is only meant for
+debugging.
+</p>
+
+<p>
+If you see this in a production setup, please remove the
```

Comment:
What's wrong with using this in production? Could be useful for monitoring.

Reply:
At the moment it wouldn't hurt. I'm not sure in which direction this is going. E.g. if we add the possibility to modify the internal state (or inject new 'faked' transfer events), it's probably better to turn if off.
It's purpose at this stage is mainly to help implementing the end2end tests, so that's why I added that message.